Add more content to abstract review system to update process (#166)
rladies/rladiesguide: content/comm/abstract-review/index.en.md

weight: 105
---

> R-Ladies Global has a system for review and feedback of conference abstracts and funding opportunities for both R-related and domain-specific conferences. The aim is to encourage women and toher gender minorities to submit proposals and improve chances of having work or applications selected, and international reviewers provide constructive feedback on areas including topics and pitch, phrasing, and subject matter.

Author: ```suggestion
R-Ladies Global has a system for review and feedback of conference abstracts and funding opportunities for both R-related and domain-specific conferences. The aim is to encourage women and other gender minorities to submit proposals and improve chances of having work or applications selected, and international reviewers provide constructive feedback on areas including topics and pitch, phrasing, and subject matter.
```

### Coordination

The requests and volunteers are tracked in AirTable: https://airtable.com/appJadVolZxoDGSIK?

> Current volunteers (Nic and Saya) alternate managing reviews on a monthly basis with Nic on even numbered months and Saya on odd-numbered months.

Reviewer: I'm not a big fan of hardcoding names in documentation because it makes it harder to maintain over time. What do you think about mentioning that the team can decide the best way to handle requests (for example, by alternating on a monthly basis)?

Reviewer 2: I like the idea! We could maybe add a link to our website (https://rladies.org/about-us/global-team/) and/or add a general email addressing linking to the team (if that exists)

> Current volunteers (Nic and Saya) alternate managing reviews on a monthly basis with Nic on even numbered months and Saya on odd-numbered months.

Author: ```suggestion
Volunteers should decide on the best way for them to manage reviews, for example, alternating between team members on a monthly basis.
```

## Handling an incoming request

Incoming requests trigger an automated message to be posted in the #team-abstract_review channel in R-Ladies Slack.

The review handler should:

1. Check the request and the submission deadline, preferred reviewer language and gender, comments made by requestor, and that any attached documents are accessible
2. Select a relevant reviewer from the list of volunteers, incrementing the value in the "reviews requested" column by 1.  If possible, try to use volunteers who haven't done too many previous reviews.

> 3. Email the potential reviewer using the template below or similar

Author: ```suggestion
3. Email 3 potential reviewers using the template below or similar
```

Perhaps if we have 3 reviewers by default, there's less onus on us to check in as to whether they've done the review, as it's more likely at least 1 will

4. Regularly check in to make sure we're not awaiting progress - see if the reviewer has added comments to the doc, and if they don't respond to the initial email assign a new reviewer

> 5. Update the "Status" and "Reviewer Email" fields in the "abstracts" table with the relevant info

Author: ```suggestion
5. Update the "Status" and "Reviewer 1", "Reviewer 2", and "Reviewer 3" fields in the "abstracts" table with the relevant info
```

6. Update the "Reviews requested" and "Reviews completed" fields in the "volunteers" table with the relevant info

> ## Template for getting in touch with reviewer

Reviewer 2: I like the idea but maybe (for the time being) let's open a new issue and put it up for discussion there :)

Hi <reviewer name>!

> Thank you so much for offering to review abstracts through the R-Ladies review program! We received a request earlier and I wondered if you'd have time in the next week or so to take a look?

Author: ```suggestion
Thank you so much for offering to review abstracts through the R-Ladies review program! We received a request earlier and I wondered if you'd have time in the next week or so to take a look?  The conference deadline is <date>.
```

> Thank you so much for offering to review abstracts through the R-Ladies review program! We received a request earlier and I wondered if you'd have time in the next week or so to take a look?

Author: Do we want to actually explicitly mention the deadline mentioned by the requestor in case of short notice ones?

Reviewer 2: I think that’s a good point when it comes to expectation and time management
